packages/react/src/codemods/emotion-to-compiled/README.md

```diff
@@ -59,4 +59,6 @@ const Component = (props) => (
 
 ## Updating Instructions
 
-When a new version of `@compiled/react` is released, Please update `@compiled/react` and `@compiled/cli` to the same new version simultaneously.
+- Bump up `@compiled/react` to x.x.x version.
```

Comment:
Can we give an example instead, e,g:

```
When wanting to update to a later version make sure both `@compiled/cli` is being ran with the same version -
watch out for it being cached!

For example when upgrading `@compiled/react` to `v0.6.0` where you've already used the CLI,
on your next run explicitly set the version number:

npx @compiled/cli@0.6.0 --preset codemods
```

Reply:
Not sure if it should go in `@compiled/cli`. Thats just generic cli with different presets. I added this in codemod preset:
```md
Please ensure you have installed `@compiled/react` in the directory where you are running this preset.
```

We can have on website 😃